hooks/useGovernedMultiTypeAccounts.ts

```diff
@@ -0,0 +1,68 @@
+import { GovernedMultiTypeAccount } from '@utils/tokens'
+import { useCallback, useEffect, useState } from 'react'
+import useGovernanceAssets from './useGovernanceAssets'
+
+export default function useGovernedMultiTypeAccounts() {
```

Comment:
i dont get what this is about without reading the code, could this be useGovernedTokenAndMintAccounts?

Reply:
This allows us to select any type of governance (program/account/mint/token), so what about `useGovernedAllTypeAccounts`?